presto-hive/src/main/java/com/facebook/presto/hive/RcFileFileWriterFactory.java

```diff
@@ -139,8 +140,14 @@ else if (ColumnarSerDe.class.getName().equals(storageFormat.getSerDe())) {
                 });
             }
 
+            Callable<Void> outputFileDeleteRoutine = () -> {
```

Comment:
@losipiuk 
I had a discussion with @nezihyigitbasi. We agreed that having try...catch where you put it is better. Please ignore the last comment about moving the try catch.